### ISSUE TITLE
add RealmEye support for wiki and trading

### ISSUE THREAD

**Issue by Spruudel** [jakcodex/muledump#153]
Made a hacky script in tampermonkey a while ago (lost it) to open the corresponding wiki page of an item, when holding down ctrl and clicking on the item. 
Some items don't have a wiki page but every category other than consumable (12?) should be fine as far as I remember.
A similar feature for trading would be also nice; as long as you somehow know that an item is tradable, the item id should be enough.

**Comment by jakcodex** [comment]
Realmeye tends to use the game ids/text for these things too. This is a bit less ambitious than the other Realmeye work I plan to incorporate, but that means it's more reasonable to accomplish. Will check it out and see about it for 9.2 or 9.3.

Btw, are you on our Discord? If so, what's your tag?

**Comment by jakcodex** [comment]
![image](https://user-images.githubusercontent.com/28816370/37085692-4d00b7c0-21bb-11e8-806f-a391e407a6bd.png)

I guess not so hard. Will need to make some aliases to redirect certain items to the proper wiki page for consumables and similar where multiple items share a page. Will need to think of the best way to implement it. Shouldn't be too hard.

**Comment by Spruudel** [comment]
Well, now I am. Tag is Spruudel#5980

**Comment by jakcodex** [comment]
I actually failed to see you join. Just gave you milk+cookie roles (one for new feature ideas that got implemented, the other for reporting original bugs).

This request is now live in Muledump Preview. Check it out. Current mechanisms are:

Ctrl+Click a single item to open the item's Realmeye menu.

Ctrl+Click two items to open the Realmeye Offers menu.

Selecting 3 or more items will nullify the request.
